app/views/posts/_list.haml

```diff
@@ -6,7 +6,7 @@
 
 %span.time-loaded= pretty_time(DateTime.now.in_time_zone)
 
-%table{style: local_assigns[:style].to_s}
+%table{style: local_assigns[:style].to_s, class: local_assigns[:table_class].to_s}
```

Comment:
You can take out the style one, grep says it's only icons and users using it. (Using `ag 'partial.*list.*style'`)